src/ff8_data.h

```diff
@@ -210,6 +210,14 @@ void ff8_find_externals()
 	ff8_externals.sub_469640 = get_relative_call(ff8_externals.pubintro_init, 0xD8);
 	ff8_externals.sub_46DBF0 = get_relative_call(ff8_externals.sub_469640, 0x5D);
 	common_externals.directsound = (IDirectSound**)get_absolute_value(ff8_externals.sub_46DBF0, 0x26);
+	common_externals.play_midi = 0x46C290;
+	common_externals.stop_midi = 0x46C6A0;
+	common_externals.midi_status = 0x46B960;
+	//common_externals.pause_midi = 0x46F2F0;
+	//common_externals.restart_midi = 0x46C670;
```

Comment:
Are these values safe enough for all the languages? Can we use a relative logic to get them otherwise?

Reply:
Safe for french only! And yes :)

Reply:
I would appreciate if we can get a relative log. Let me know if I can help you there in case! Thanks :D

Reply:
This was actually really hard to do, but here we go :)